src/EventListener/ResourceIndexListener.php

```diff
@@ -39,7 +39,12 @@ public function updateIndex(GenericEvent $event): void
 
         foreach ($this->persistersMap as $objectPersisterId => $modelClass) {
             if ($resource instanceof $modelClass) {
-                $this->resourceRefresher->refresh($resource, $objectPersisterId);
+                // Respect an enabled-switch on the resource
+                if (method_exists($resource, 'isEnabled') && !$resource->isEnabled()) {
```

Comment:
ooor we can change it to ToggleableInterface instance :)

Reply:
Yeah, I will have a look at that soon, I hope. But its the better solution, I agree. This one was born out of necessity :)

Reply:
@kortwotze, would you find some time to change it? :)